api/pb/types.proto

```diff
@@ -22,6 +22,6 @@ message PinOptions {
   sint32 ReplicationFactorMax = 2;
   string Name = 3;
   uint64 ShardSize = 4;
-  repeated string UserAllocations = 5;
+  reserved 5;
```

Comment:
I suggest adding a comment stating that this field is reserved for `UserAllocations` and not just a general reserved field for possible future needs.